mzLib/FlashLFQ/FlashLFQResults.cs

```diff
@@ -842,6 +844,7 @@ internal void RevisedModifiedPeptides()
                         var allSeq = isoPeptidePeaks
                             .Where(p => p != null)
                             .SelectMany(p => p.Identifications)
+                            .Where(p=>p.BaseSequence == originalPeptide.BaseSequence)// do not output the peptide with different base sequence in the peptide result
                             .Select(p => p.ModifiedSequence)
                             .Distinct()
```

Comment:
We still need to modify the peptides with the different base sequence by setting it to ambiguous/intensity to zero